sw/airborne/subsystems/ahrs/ahrs_int_cmpl_quat.c

```diff
@@ -270,7 +270,11 @@ void ahrs_icq_update_accel(struct Int32Vect3 *accel, float dt)
 #define ACC_FROM_CROSS_FRAC INT32_RATE_FRAC + INT32_SPEED_FRAC - INT32_ACCEL_FRAC - COMPUTATION_FRAC
 
     const struct Int32Vect3 vel_tangential_body =
+#ifdef VERTICAL_WING_HYBRID
+      {0, 0, -(ahrs_icq.ltp_vel_norm >> COMPUTATION_FRAC)};
```

Comment:
can you add some comments about the plane orientation in this case ? is there a way to make this more generic ?